src/main/java/com/hedvig/claims/config/AxonConfiguration.java

```diff
@@ -17,7 +18,8 @@
   public EventUpcasterChain eventUpcasters() {
     return new EventUpcasterChain(
       new ClaimCreatedEvent_v1(),
-      new PaymentAddedEvent_v1()
+      new PaymentAddedEvent_v1(),
+      new ClaimFileUploadedEventUpcaster()
```

Comment:
For consistency I guess

```suggestion
      new ClaimFileUploadedEventUpcaster_v1()
```